huaweicloud/services/acceptance/cts/resource_huaweicloud_cts_tracker_test.go

```diff
@@ -0,0 +1,142 @@
+package deprecated
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	cts "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/cts/v3/model"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+)
+
+func TestAccCTSTracker_basic(t *testing.T) {
+	rName := acceptance.RandomAccResourceNameWithDash()
+	resourceName := "huaweicloud_cts_tracker.tracker"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckCTSTrackerDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCTSTracker_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCTSTrackerExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "bucket_name", rName),
+					resource.TestCheckResourceAttr(resourceName, "file_prefix", "cts"),
+					resource.TestCheckResourceAttr(resourceName, "lts_enabled", "true"),
+					resource.TestCheckResourceAttr(resourceName, "status", "enabled"),
+					resource.TestCheckResourceAttr(resourceName, "type", "system"),
+				),
+			},
+			{
+				Config: testAccCTSTracker_update(rName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "bucket_name", rName),
+					resource.TestCheckResourceAttr(resourceName, "file_prefix", "cts-updated"),
+					resource.TestCheckResourceAttr(resourceName, "lts_enabled", "false"),
+					resource.TestCheckResourceAttr(resourceName, "status", "disabled"),
+					resource.TestCheckResourceAttr(resourceName, "type", "system"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccCTSTrackerImportState(resourceName),
+			},
+		},
+	})
+}
+
+func testAccCTSTrackerImportState(name string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		tracker, ok := s.RootModule().Resources[name]
+		if !ok {
+			return "", fmt.Errorf("CTS tracker not found")
+		}
+
+		name := tracker.Primary.Attributes["name"]
+		if name == "" {
+			return "", fmt.Errorf("CTS tracker resource %s not found", tracker.Primary.ID)
+		}
+		return name, nil
+	}
+}
+
+func testAccCheckCTSTrackerDestroy(s *terraform.State) error {
+	return nil
+}
```

Comment:
destroy should test whether the tracker state is `disabled`.

Reply:
when the system tracker is destroyed, the status just keep it as is